apicode/newapi.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import inquirer
+import textwrap
+import os
+from inquirer.themes import GreenPassion
+
+
+class NewsApi:
+    # function for picking the news source from the user
+    def get_user_source(self):
+        questions = [
+            inquirer.List('sources',
+                          message="From which news source would you like to acquire your headlines?",
+                          choices=['nbc-news', 'cnbc',
+                                   'bbc-news', 'cnn'],
+                          ),
+
+
+        ]
+        # theme adds a green background to the items in the list
+        answers = inquirer.prompt(questions, theme=GreenPassion())
+        print(answers['sources'])
+
+        return answers['sources']
+    # function to retrieve a product key for the environment variable
+
+    def add_api_key(self):
```

Comment:
This method name is annoying. It would be better if you could name it `get_api_key`